tests/integration/suite/test_workloads.py

```diff
@@ -343,7 +343,9 @@ def _timestamp_reset():
 
 def test_perform_workload_action_read_only(admin_mc, admin_pc, remove_resource,
                                            user_mc, user_factory):
-    """Tests workload actions with a read-only user and a member user."""
+    """Tests workload actions with a read-only user and a member user.
+       NOTE: if backporting a change to 2.4 and below that this test
+       encounters, ensure that the change works manually."""
```

Comment:
Wording is slightly confusing, how about: 

> Note: this test exists only in 2.5+ due to pod scheduling on the local cluster. Ensure manually that any changes in 2.4 and below work.

Reply:
Agreed. Done.